src/web/page/templates.rs

```diff
@@ -332,9 +332,12 @@ impl tera::Filter for IconType {
         let icon_file_string = font_awesome_as_a_crate::svg(type_, &icon_name[..]).unwrap_or("");
 
         let icon = format!(
-            r#"<span {class} aria-hidden="{aria_hidden}"{aria_label}{id}>{icon_file_string}</span>"#,
+            "<span \
+                class=\"{class} {class_extra}\" \
```

Comment:
What happens for `class=" "`? Does the browser just ignore it?

Reply:
yep

Reply:
It's just an attribute with no values. :)